src/lxc/utils.h

```diff
@@ -427,8 +427,34 @@ extern int lxc_strmunmap(void *addr, size_t length);
 /* initialize rand with urandom */
 extern int randseed(bool);
 
-inline static bool am_unpriv(void) {
-	return geteuid() != 0;
+inline static bool am_unpriv(void)
```

Comment:
Can we call this "am_host_unpriv()" or something, to disambiguate from "privileged in a user namespace" ?

Reply:
Sure, I'll push that change shortly.